lib/net.js

```diff
@@ -1389,7 +1389,7 @@ Server.prototype.listen = function() {
 };
 
 function lookupAndListen(self, port, address, backlog, exclusive) {
-  require('dns').lookup(address, function(err, ip, addressType) {
+  require('dns').lookup(address, function emitDnsLookup(err, ip, addressType) {
```

Comment:
It was under dns context, however it should be a listen or doListening. I will modify on next commit :) @lpinca